tests/collections/nlp/test_dialogue.py

```diff
@@ -276,3 +277,42 @@ def test_dialogue_nearest_neighbour_mean_pooling():
     assert torch.equal(
         torch.ones(8, 768).float() * 0.5, DialogueNearestNeighbourModel.mean_pooling(model_output, attention_mask)
     )
+
+@pytest.mark.unit
+def test_dialogue_get_entity_embedding_from_hidden_states():
```

Comment:
I think you should have at least 2-3 test cases for each function you're testing. Preferably, they should include corner cases to show how your function is robust to various possible input

Reply:
Future: shorter test case cover one concept to test instead of long test case (too many concept to test per test case)